test/taskincludes_2_7_style.yml

```
@@ -0,0 +1,9 @@
+---
+- hosts: webservers
+  vars:
+    varset: varset
+  tasks:
+    - include_tasks:
```

Also test negative cases with invalid values
```suggestion
    - include_tasks:
    - include_tasks:
```